CHANGELOG.md

```diff
@@ -28,9 +28,18 @@
 ### Fixed
 - Fixed not able to connect to new Technic Move hub with `LWP3Device()`.
 - Removed `gc_collect()` from `tools.run_task()` loop to fix unwanted delays.
+- Fixed `await wait(0)` never yielding, so parallel tasks could lock up ([support#1429]).
+
+### Removed
+- Removed `loop_time` argument to `pybricks.tools.run_task` as this wasn't
+  having the desired effect, and would cause loop round trips to take `10 ms`
+  for every `await wait(1)` ([support#1460]). Passing the argument is still
+  allowed for backwards compatibility, but it won't do anything.
```

Comment:
Looks like we missed fixing up the changelog for removing loop_time completely.

Reply:
Thanks, will fix as part of revision in #254